### DISCUSSION TITLE
Projects don't load on Android

### DISCUSSION THREAD

**Discussion by gepbird** [community/community#109320] [Mobile]
Tried updating and restarting the app.

Screenshot with error code:

> ![Screenshot_20240220-085423_GitHub_1_1.png](https://github.com/community/community/assets/29818440/672ab6b3-2753-45b6-ad75-0199366b8b0e)

**Comment by matt400** [comment]
Yeah, I have the same error. It started appearing when I have added multiple drafts.

**Comment by karinzaa** [comment]
I have the same error and has this message.

Something went wrong while executing your query. Please include `B200:8A28C:6160D6:680FFB:65D4871D` when reporting this issue.

**Comment by Kingsize072** [comment]
Either me, I got an error like this.

![1708427712929.jpg](https://github.com/community/community/assets/101883238/7b7b5ffd-1942-4e03-8aef-ce1fe73e2137)

**Comment by iskae** [comment]
Hi folks 👋 

Thank you all for your reports ❤️ We are currently investigating this and we will get back to you as soon as possible 🙇

**Comment by koeppel** [comment]
Also when trying to add a project to an issue the projects that are linked to the given repository are not displayed.
